.cirrus.yml

```diff
@@ -1,18 +1,38 @@
-container:
-  image: gcr.io/flutter-cirrus/build-flutter-image:latest
+# CIRRUS CONFIGURATION FILE
```

Comment:
This looks SOOO much better.  Note, however, that I didn't check to make sure that this PR doesn't accidentally drop any shards that we were running before.